src/mulsi/wrapper.py

```diff
@@ -0,0 +1,20 @@
+"""Abstract class for MULSI models.
+"""
+
+from abc import ABC, abstractmethod
+
+
+class MulsiWrapper(ABC):
```

Comment:
How will you apply this class to an existing model from HF ? From what I understand you have to redefine the `compute_loss()` function, I can't really see how it can be used

Reply:
See previous comment.